apps/studio/styles/globals.css

```
@@ -1,13 +1,113 @@
+
+@keyframes sway {
+  0%,
+  100% {
+    transform: rotate(-10deg) scale(1.5) translateY(4rem);
+  }
+  50% {
+    transform: rotate(10deg) scale(1.5) translateY(2rem);
+  }
+}
+
+@import 'config/typography.css';
```

Stylelint (17.10.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)